docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,60 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
+
+For the first run, you need these commands:
+
+```bash
+./prepare-workspace.sh
+./ci/build-test.sh
+```
+
+To test the code generator or docs linter tool after modifying them, you need to run this command:
+
+```bash
+./ci/build-test.sh
+```
+
+To test the Unity project after modifying it, you need to run this command:
+
+```bash
+./ci/test.sh
+```
+
+## Unity tests
+
+For Unity tests, you can also run tests from the Unity Test Runner Window.
+
+### 1. Open the Unity project in Unity 
+
+The unity project can be found in this directory within the repository: `/worker/unity`.
+
+### 2. Open test runner window
+
+From the toolbar at the top of the Unity Editor, select `Window > Test Runner`.
+
+### 3. Run tests
```

Comment:
Shouldn't this section link to Unity's docs on using the test runner? Why are we duplicating information?

Reply:
I'll add a link, but I'm keeping the contents here to prevent requiring the readers to hop document to document.